index.js

```diff
@@ -72,21 +72,26 @@ export const createConfig = async({
   
   authClient = new OktaAuth(oktaAuthConfig);
 
+  const connectionTimeout = httpConnectionTimeout ?? 15;
```

Comment:
set default value? 
httpConnectionTimeout || 15;

Reply:
nit: set the default value w/o creating a new variable
httpConnectionTimeout = httpConnectionTimeout || 15;